cogs/displayinfo.py

```diff
@@ -0,0 +1,116 @@
+import discord
+import constantvariables
+import mongoDB
+from discord.ext import commands
+from discord.ext import tasks
+from datetime import datetime
+from datetime import date
+import datetime as dt
+from pymongo.mongo_client import MongoClient
+from pymongo.server_api import ServerApi
+import asyncio
+
+
+
+
+# Create a new client and connect to the MongoDB server
+mongoClient = MongoClient(mongoDB.MONGODB_URI, server_api=ServerApi('1'))
```

Comment:
What is the API version 1 for? Can we only support API version 1? How do we upgrade API version to support newer commands?

Reply:
Apparently, there is currently only one API version available and that is API version 1 which should already support newer commands.